source/docs/magento2/graphql.md

```diff
@@ -0,0 +1,66 @@
+---
+id: magento2-graphql
+title: Magento2 GraphQL schema
+---
+
+Front-Commerce aims at making Magento developers productive by allowing them to work with what they know from Magento.
+
+In order to achieve this, our goal is to expose as much [Magento’s GraphQL schema](https://devdocs.magento.com/guides/v2.3/graphql/) as possible in Front-Commerce.
+We follow evolutions in Magento core very closely, and aim at including the new stable parts of this schema in Front-Commerce **as soon as they match the parts we’ve developed over the years in terms of feature, performance and stability**.
+
+This section details what is available so far.
+
+<blockquote class="info">
+  Please note that it relies upon the [GraphQL modules remote schema](/docs/advanced/graphql/remote-schemas.html) support from Front-Commerce.
+  We recommend to have at least a basic understanding of how it works if you know Magento’s GraphQL schema and would like to find yourself at home in Front-Commerce!
+</blockquote>
+
+## Enable Magento2 GraphQL module
+
+Front-Commerce supports both Magento 2.2 and 2.3 versions.
+For this reason, GraphQL support for Magento is not part of the default Magento2 [meta module](/docs/advanced/graphql/meta-modules.html).
+
+To enable GraphQL support for Magento 2.3+, you must register the `server/modules/magento2-graphql` module in your application.
+
+To do so, add the following line in your [`.front-commerce.js`](/docs/reference/front-commerce-js.html) file:
+
+```diff
+// .front-commerce.js
+module.exports = {
+  name: "Front-Commerce",
+  url: "https://www.front-commerce.test",
+  modules: [],
+  serverModules: [
+    { name: "FrontCommerceCore", path: "server/modules/front-commerce-core" },
+-    { name: "Magento2", path: "server/modules/magento2" }
++    { name: "Magento2", path: "server/modules/magento2" },
++    { name: "Magento2GraphQL", path: "server/modules/magento2-graphql" }
+  ]
+};
+```
+
+**Restart your server**, and you must now see the additional endpoints.
+
+## Store configurations
+
+The public store configurations are available under the `Magento2GraphQL_storeConfig` query.
+
+Example:
+
+```graphql
+{
+  Magento2GraphQL_storeConfig {
+    timezone
+  }
+}
+```
+
+See [the related Magento DevDocs page](https://devdocs.magento.com/guides/v2.3/graphql/reference/store-config.html) for more information.
```

Comment:
```suggestion
See [the related Magento DevDocs page](https://devdocs.magento.com/guides/v2.3/graphql/reference/store-config.html) to learn which configurations are exposed by default in Magento's schema.
```

Do you know if there is some doc explaining how to add new configurations in Magento? It would be great to document it here.